src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -89,8 +89,8 @@ export class WorkItemDetailComponent implements OnInit {
   getAllUsers() {
     this.userService.getAllUsers()
       .then((users) => {
-        this.users = users;
-        this.filteredUsers = users;
+        this.users = JSON.parse(JSON.stringify(users));
+        this.filteredUsers = JSON.parse(JSON.stringify(users));
```

Comment:
The problem with this approach is that it doesn't preserve the types. I think we may want cloneDeep<T>() from lodash.